uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/modals/leave-request.js

```diff
@@ -0,0 +1,51 @@
+var modal = require('./modal');
+
+module.exports = (function () {
+  return modal.extend({
+
+    /**
+     * [selectTab description]
+     * @return {[type]} [description]
+     */
+    selectTab: function (tabName) {
+      var casper = this.casper;
+
+      casper.then(function () {
+        casper.click('div.chr_leave-request-modal__tab li[heading=' + tabName + '] a');
+      });
+
+      return this;
+    },
+
+    /**
+     * [addCommentToScope description]
+     * @return {[type]} [description]
+     */
+    addCommentToScope: function (comment) {
```

Comment:
Function name should just be `addComment` and in the description in can be mentioned that we are adding to scope. But `addCommentToScope` is confusing.